src/editor/objects/Model.js

```diff
@@ -70,7 +70,19 @@ export default class Model extends Object3D {
     return clipOptions;
   }
 
-  getActiveClipIndices() {
+  getActiveItems(indices) {
+    if (this.model && this.model.animations) {
+      return indices
+        .filter(item => item >= 0 && this.model.animations[item])
+        .map(item => {
+          const clip = this.model.animations[item];
+          return { label: clip.name, value: item };
+        });
+    }
+    return null;
```

Comment:
Should this return an empty array?

Reply:
React seems to handle that case but I have updated to avoid confusion.